pkg/server/server.go

```diff
@@ -68,6 +70,17 @@ var (
 		NativeHistogramMaxBucketNumber:  100,
 		NativeHistogramMinResetDuration: time.Hour,
 	}, []string{"grpc_service", "grpc_method"})
+
+	latencyHistogramName = "latency"
```

Comment:
I am open to making this more specific to check latency only.  However, I envision we may want to have something similar for list object which makes me want to make this more flexible.

Reply:
latency is super generic. I'd recommend a more specific metric name such as `request_duration_by_query_count_ms`. That way the name is a little more clear and indicates the unit of the underlying value as well.

Reply:
If we are moving to request_duration_by_query_count_ms, we should also update the flag/config from latencyDBQueryCountBuckets to requestDurationDatastoreQueryCountBuckets